dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -653,14 +653,17 @@
 
         if (data["contentletIdentifier"]) {
             if (ngEditContentletEvents) {
-                ngEditContentletEvents.next({
+                // This is needed to wait for re-index on update, since we are saving with ContentletAjax.saveContent
+                setTimeout(()=>{
+                    ngEditContentletEvents.next({
                     name: 'save',
                     data: {
                         identifier: data.contentletIdentifier,
                         inode: data.contentletInode,
                         type: null
                     }
                 });
+                },3000);
```

Comment:
what happend if the reindex take more time?

Reply:
we don't refresh the content, this just covers more scenarios, not the ideal fix. But is what we need to deal while keep using `ContentletAjax.saveContent`